src/elements/button/src/stories.tsx

```diff
@@ -81,7 +89,7 @@ export const SecondaryVariant = () => (
 )
 
 export const PrimaryDisabledVariant = () => (
-  <Button variant="primary" disabled>
+  <Button variant="hollow" disabled>
```

Comment:
mentioned irl, seems like this has overlap with the work josh is doing on inverse button states